aea/context/base.py

```diff
@@ -43,7 +43,6 @@ def __init__(
         ownership_state: OwnershipState,
         preferences: Preferences,
         goal_pursuit_readiness: GoalPursuitReadiness,
-        task_queue: Queue,
```

Comment:
there's no queue for task manager anymore - tasks can be enqueued by using the method `enqueue_task`

Reply:
it is a step in the opposite direction of the actor model. However, until that, this is a much simpler solution. We don't need the 'dispatch' thread anymore.